components/dialog/test/dialog-ifrau.visual-diff.html

```diff
@@ -2,57 +2,68 @@
 <html lang="en">
 <head>
 	<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
-	<script>
-		const preferNative = (window.location.search.indexOf('preferNative=false') === -1);
-		window.D2L = { DialogMixin: { preferNative: preferNative } };
-	</script>
+	<title>d2l-dialog-ifrau</title>
 	<script type="module">
 		import '../../typography/typography.js';
-		import '../../button/button.js';
-		import '../dialog.js';
 	</script>
-	<title>d2l-dialog-ifrau</title>
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<meta charset="UTF-8">
 </head>
 <body class="d2l-typography">
-	<d2l-button id="open">Open Dialog</d2l-button>
-
-	<d2l-dialog id="ifrau-dialog" title-text="Title" width="400">
-		<div>Line 1</div>
-		<div>Line 2</div>
-		<div>Line 3</div>
-		<div>Line 4</div>
-		<d2l-button slot="footer" primary>Yes</d2l-button>
-		<d2l-button slot="footer" id="cancel">No</d2l-button>
-	</d2l-dialog>
-
-	<script>
+	<div id="ifrau-dialog-container" style="width: 650px"></div>
```

Comment:
Technically this isn't necessary. We can give the iframe an ID via ifrau and emit events from that element. We can also just set the iframe's parent as the HTML body. I think I'm going to do that.

Reply:
Oh... Except without that the page width gets a bit wonky and things end up looking kind of gross. I'll still set the ID on the iframe because it makes some lookups easier in the visual diff tests themselves, but I think I'll keep the parent div.

Reply:
Eh, except then we would need to switch over the iframe to emit all of the events anyways, or the test would need to know both about the div and the iframe. Maybe this isn't worth doing.